src/components/app/App.js

```diff
@@ -29,6 +31,10 @@ export default function App({ initialActivities }) {
   const filteredActivities =
     filterActivities(activities, searchTerm) || activities
 
+  const bookmarkedActivities = activities.filter(
+    activity => activity.isBookmarked === true
+  )
+
```

Comment:
This will save your updated activities array each time your activities are updated. This way you don't need to write it in every function :)
```suggestion
useEffect(() => {
      saveToLocal('localActivities', activities)
}, [activities])
```

Reply:
Reworked this. Thanks a lot.